$OpenBSD: patch-config_tests_unix_compile_test,v 1.1.1.1 2016/12/25 14:13:11 zhuk Exp $
Pass -Wl, flags to linker as well.
Unbreaks, e.g., PulseAudio config test.
--- config.tests/unix/compile.test.orig	Tue Dec 13 01:09:38 2016
+++ config.tests/unix/compile.test	Tue Dec 13 01:10:07 2016
@@ -40,7 +40,7 @@ while [ "$#" -gt 0 ]; do
         CFLAGS="$CFLAGS \"$PARAM\""
         CXXFLAGS="$CXXFLAGS \"$PARAM\""
         ;;
-    -L*|-l*|-pthread)
+    -L*|-l*|-Wl,*|-pthread)
         LFLAGS="$LFLAGS \"$PARAM\""
 	;;
     -I*)
